index.html

```diff
@@ -48,5 +48,193 @@ <h1 class="title">I’m Mariane<span id="info" class="title">Glad to see you!</s
         
     </section>
     
+    <section class="works-section">
+        <div class="grid-container">
+            <ul>
+                <li class="project">
+                    <img src="src/Snapshoot-Portfolio.png" alt="Snapshot">
```

Comment:
- [ ] You did a great job implementing cards but as the previous reviewer said you have to follow [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and fix the issue with cards. Here you have a screenshot of how I am seeing cards:
![image](https://user-images.githubusercontent.com/74506933/153707827-cc115204-594e-45c3-ad3f-738cffa0d4bd.png)